test/fixtures/user/user.js

```diff
@@ -267,12 +267,6 @@ module.exports = () => {
       linkedin_id: "testuser1",
       github_id: "testuser1",
       github_display_name: "Test User",
-      phone: "1234567890",
-      email: "tu@gmail.com",
-      chaincode: "1234",
-      tokens: {
-        githubAccessToken: "githubAccessToken",
-      },
```

Comment:
why are we removing these fields. I know these are confidential but they still needed to be added in the DB. Right ? then while accessing we will remove this data.